htdocs/core/lib/functions.lib.php

```diff
@@ -828,7 +828,7 @@ function checkVal($out = '', $check = 'alphanohtml', $filter = null, $options =
 				// We replace chars from a/A to z/Z encoded with numeric HTML entities with the real char so we won't loose the chars at the next step (preg_replace).
 				// No need to use a loop here, this step is not to sanitize (this is done at next step, this is to try to save chars, even if they are
 				// using a non coventionnel way to be encoded, to not have them sanitized just after)
-				$out = preg_replace_callback('/&#(x?[0-9][0-9a-f]+;?)/i', 'realCharForNumericEntities', $out);
+				$out = preg_replace('/&#(x?[0-9][0-9a-f]+;?)/i', 'realCharForNumericEntities', $out);
```

Comment:
preg_replace_callback and preg_replace
preg_replace replace string with the value 'realCharForNumericEntities'
preg_replace_callback repalce string with the result of the function realCharForNumericEntities()

Reply:
more info on the french forum link